### PR TITLE
Added FLLC for fixed wing and FAST turbines

### PR DESCRIPTION
**Pull request by tonyinme**
This is the implementation of the filtered lifting line correction for the fixed wing.
I still need help to implement the turbine fast version.

### CODE REVIEW

File: amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H [hunk 1]
```diff
@@ -330,6 +331,14 @@ struct UpdatePosOp<TurbineFast, SrcTrait>
             vel_pos[i].y() = static_cast<amrex::Real>(pyvel[i]) + bp.y();
             vel_pos[i].z() = static_cast<amrex::Real>(pzvel[i]) + bp.z();
         }
+
+        auto& meta = data.meta();
+        if (!meta.fllc_use || !meta.fllc_is_init) return;
+
+        // Implement the FLLC
+        FLLCOpTurbineBase<TurbineFast, SrcTrait> op(data);
```
Comment (psakievich) [inline amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H:339]: we should think about where this operator should live/how it is instantiated a little more.  Here you are reconstructing it at every timestep.  This makes it harder to cache the correction and apply it across timesteps like we did in nalu-wind.

**Review by psakievich — COMMENTED** [review]
Some initial thoughts

File: amr-wind/wind_energy/actuator/CMakeLists.txt [hunk 2]
```diff
@@ -4,9 +4,10 @@ target_sources(${amr_wind_lib_name}
   actuator_utils.cpp
   Actuator.cpp
   ActuatorContainer.cpp
+  FLLCOpBase.H
```
[the remark this answers is not in the document]
Comment (tonyinme) [reply amr-wind/wind_energy/actuator/CMakeLists.txt:7]: good point! fixed.

**Comment by psakievich** [comment]
@tonyinme I think I've done everything but compute the relative velocity for the turbines.

**Comment by psakievich** [comment]
@michaeljbrazell and @tonyinme i believe this is done from my end.

**Comment by tonyinme** [comment]
I added the documentation. 

This PR is ready. 

@michaeljbrazell let us know if you have any suggestions.
If not, it can be merged.

**Review by michaeljbrazell — APPROVED** [review]

**Comment by psakievich** [comment]
Tested unit tests on the GPU build and a couple regression tests. Didn't have gold files, but regression tests ran to completion.

**Pull request merged**